test/unit/component/parser/json/BpmnJsonParser.sequenceFlow.default.test.ts

```diff
@@ -77,144 +78,37 @@ describe('parse bpmn as json for default sequence flow', () => {
     });
   });
 
-  it('json containing one process with a sequence flow defined as default in a exclusive gateway', () => {
+  each([
```

Comment:
:+1: very cool feature, simple and consice
:question: just to let me know: was the jest bump version required or did you just take the opportunity to upgrade it as part of the `jest-each` lib introduction

Reply:
I can not use the last version of jest & jest-each (26.0.x), because is not supported by the last version of ts-jest.
But, I update jest & take the last version of jest-each supported by ts-jest, to have the last fixes of the libraries, as the upgrades don't introduce problem in our tests :wink: